ziit/umbrel-app.yml

```
@@ -0,0 +1,21 @@
+manifestVersion: 1
```

Use manifest version 1.1 to enable hooks.
```suggestion
manifestVersion: 1.1
```